### PR TITLE
Remove final from render

### PR DESCRIPTION
**Pull request by deanmcpherson**
Can we please remove final from the render method? 

I'm playing with adding support for rendering react/vue components inside of volt components with https://github.com/ijpatricio/mingle, but naturally it needs to be able to hijack the render method to function.

I can't think of any reasons why removing the final keyword would be a breaking change here.

### CODE REVIEW

File: src/ComponentFactory.php [hunk 1]
```diff
@@ -79,9 +79,11 @@ protected function requirePath(string $path): void
                         : $variable;
                 }, get_defined_vars());
             })();
+            CompileContext::instance()->variables =  array_merge(CompileContext::instance()->variables, $variables);
```
Comment (taylorotwell) [inline src/ComponentFactory.php:82]: Why is this unrelated change made?

Comment (deanmcpherson) [reply src/ComponentFactory.php:82]: @taylorotwell sorry, yep this was needed further down the track of my implementation. The mingle lib needs a `component` method defined to point to the react/vue component it should render, and I wanted to add it automatically as I'm extracting the component from the blade file itself  (see https://github.com/deanmcpherson/mingle/blob/main/src/Mingle.php), but variables added to the default array before this is run were getting overridden here.

File: src/VoltServiceProvider.php [hunk 2]
```diff
@@ -116,7 +116,7 @@ protected function registerTestingMacros(): void
                 $component = FragmentMap::get($component);
             }
 
-            return $this->assertSeeLivewire($component); // @phpstan-ignore-line
+            return $this->assertSeeLivewire($component);
```
Comment (taylorotwell) [inline src/VoltServiceProvider.php:119]: Why was this change made?

**Comment by deanmcpherson** [comment]
The static analysis threw an error because there was nothing to ignore
anymore. I suspect the original suppression was because of the use of
final? Happy to revert if not

File: src/VoltServiceProvider.php (hunk 2 above)
Comment (deanmcpherson) [reply src/VoltServiceProvider.php:119]: The static analysis in the github action for the PR threw an error because there was nothing to ignore anymore. I suspect the original suppression was because of the use of final? Happy to revert
